data_model/tests/data_model.rs

```diff
@@ -302,3 +302,18 @@ fn cannot_forbid_minting_on_asset_mintable_infinitely() -> Result<(), ParseError
     assert!(definition.forbid_minting().is_err());
     Ok(())
 }
+
+#[test]
+fn account_id_parsing() -> Result<(), ParseError> {
+    // `AccountId` should have format `name@domain_name`
+    let account_normal: AccountId = "test@hello".parse()?;
+    assert_eq!(account_normal.name.to_string(), String::from("test"));
```

Comment:
I would suggest to do the following: 
```suggestion
    assert_eq!(account_normal.name.as_ref(), "test");
```